src/overmap.h

```diff
@@ -294,14 +295,12 @@ class overmap
         // parse data in an old overmap file
         void unserialize_legacy( std::istream &fin );
         void unserialize_view_legacy( std::istream &fin );
+        const city &get_nearest_city( const tripoint &p ) const;
```

Comment:
Instead of making this public, please see if you can use overmapbuffer::closest_city()